bookkeeper-server/src/test/java/org/apache/bookkeeper/client/SlowBookieTest.java

```diff
@@ -419,11 +419,9 @@ public void testWaitForWritable() throws Exception {
             final long timeout = 10000;
 
             // waitForWritable async
-           new Thread(() -> {
-                isWriteable.set(lh.waitForWritable(writeSet, 0, timeout));
-            }).start();
-            TimeUnit.MILLISECONDS.sleep(5000);
-            assertFalse(isWriteable.get());
+            new Thread(() -> isWriteable.set(lh.waitForWritable(writeSet, 0, timeout))).start();
+
+            Awaitility.await().untilAsserted(() -> assertFalse(isWriteable.get()));
```

Comment:
```suggestion
            Awaitility.await().pollDelay(5, TimeUnit.SECONDS).untilAsserted(() -> assertFalse(isWriteable.get()));
```

Because the isWritable is false from the beginning, we want to wait a moment to verify it still in `waitForWritable` backoff